### ISSUE TITLE
Thank you for your excellent work. Are all 3D models watertight?

### ISSUE THREAD

**Issue by 2017211801** [allenai/objaverse-xl#41]
Thank you for your excellent work. Are all 3D models watertight?

**Comment by nallath** [comment]
No, they are just scraped from all over the internet. There is no guarantee that they are.

**Comment by 2017211801** [comment]
Are special objects such as clothes open mesh or closed mesh, i.e., do they have open necklines and cuffs?
